### ISSUE TITLE
Failed to import Python classes at startup

### ISSUE THREAD

**Issue by lassoan** [Slicer/Slicer#7579]
## Summary

While instantiating  `ArterialCalcificationPreProcessor` module in VMTK extension, this error is logged:

```
Failed to load vtkSlicerCrossSectionAnalysisModuleLogicPython: No module named vtkSlicerShapeModuleMRMLPython
```

Printing the stack trace in `importClassesFromDirectory` in `util.py` indicates that the issue is that `vtkSlicerCrossSectionAnalysisModuleLogicPython` module logic failed to load because vtkSlicerShapeModuleMRMLPython.pyd file, which is provided by the "ExtraMarkups" extension was not found.

```
Traceback (most recent call last):
  File "C:\D\S4D\Slicer-build\bin\Python\slicer\util.py", line 168, in importClassesFromDirectory
    importModuleObjects(from_module_name, dest_module_name, type_info)
  File "C:\D\S4D\Slicer-build\bin\Python\slicer\util.py", line 206, in importModuleObjects
    module = imp.load_module(from_module_name, fp, pathname, description)
  File "C:\D\S4D\python-install\Lib\imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "C:\D\S4D\python-install\Lib\imp.py", line 342, in load_dynamic
    return _load(spec)
  File "<frozen importlib._bootstrap>", line 711, in _load
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 565, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 1173, in create_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
ImportError: Failed to load vtkSlicerCrossSectionAnalysisModuleLogicPython: No module named vtkSlicerShapeModuleMRMLPython
```

The issue is that vtkSlicerShapeModuleMRMLPython.pyd is in 

c:\D\S4D\Slicer-build\slicer.org\Extensions-30987\ExtraMarkups\lib\Slicer-5.7\qt-loadable-modules\vtkSlicerShapeModuleMRMLPython.pyd

but  this folder is not in the PYTHON path.

## Steps to reproduce

- Install VMTK Slicer extension (install the required ExtraMarkups extension as well)
- Restart Slicer => this error is logged: `Failed to load vtkSlicerCrossSectionAnalysisModuleLogicPython: No module named vtkSlicerShapeModuleMRMLPython`

## Environment
- Slicer version: Slicer-5.7.0. 2024-02-08
- Operating system: Windows11 (probably all OS)

**Comment by dzenanz** [comment]
I am experiencing this issue in 5.6.2. Is this supposed to be fixed in that version?
<img width="1443" alt="Screenshot 2024-05-21 13 09 27" src="https://github.com/Slicer/Slicer/assets/1792121/7f504870-d50c-41ee-9a5a-1e17bc8e9e5b">

**Comment by dzenanz** [comment]
Today's nightly does not seem to have VMTK extension. Is this expected?

**Comment by dzenanz** [comment]
> Today's nightly does not seem to have VMTK extension. Is this expected?

Oops, I was looking at an older nightly, from 2024-04-23. Tonight's nightly also has the loading issue:
<img width="1441" alt="Screenshot 2024-05-21 13 20 42" src="https://github.com/Slicer/Slicer/assets/1792121/707524b7-6e2c-4a5e-9117-9d409b22922d">

**Comment by lassoan** [comment]
That is all good, it is a false alarm - see https://github.com/vmtk/SlicerExtension-VMTK/issues/22
We could probably tweak the build system to install the VMTK Python package in a subdirectory, but this issue will be resolved anyway when VMTK will be distributed on PyPI as a VTK remote module.

**Comment by jcfr** [comment]
As a side note, we haven't yet transferred build of stable extensions onto the new build machine. 

This means, updated extensions are not yet published in the context of Slicer 5.6.2
